src/components/bc-connector-list.ts

```diff
@@ -27,9 +29,27 @@ export class ConnectorList extends withTwind()(BitcoinConnectElement) {
       connectors.push(html`<bc-lnc-connector></bc-lnc-connector>`);
     }
 
-    return html` <div class="flex justify-center items-start flex-wrap gap-5">
-      ${connectors}
-    </div>`;
+    return html`
+      <div class="flex justify-center items-start flex-wrap gap-5">
+        ${connectors}
+      </div>
+      <div class="flex flex-col items-center w-full font-sans text-sm">
```

Comment:
this should be moved to a different component I think, because this component is only about showing a list of connectors.